src/app/app.component.scss

```diff
@@ -148,4 +163,17 @@ div.dragged {
     svg.preview line {
         display: none;
     }
+
+    text.tick {
+        fill: #595959;
+        text-anchor: end;
+        font-size: 0.7px;
+        /* Prevent accidental selection of axis markers on drag */
+        cursor: default;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
```

Comment:
`user-select: none;` is enough here, the angular-cli automatically adds the missing vendor-specific properties.